GH #286 Updated Cart Price Rule topic (#299)
magento/merchdocs: src/marketing/price-rules-cart-create.md

        | Yes | Applies the discount amount separately to the subtotal and shipping amounts. |
        | No | Applies the discount amount only to the subtotal. |

    - To stop processing other rules after this rule is applied, set **Discard Subsequent Rules** (![Option toggle]({% link images/images/btn-switch-yes.png %})) to `Yes`. This setting prevents multiple discounts from being applied to the same product.

         | Yes | Prevents any other pricing rules that may apply to a product from being applied. Discarding subsequent rules means that, in the event that multiple pricing rules apply to the same product, only the pricing rule with the highest defined priority (in a rule's Priority field) will be applied to the qualifying product. This prevents multiple pricing rules from stacking and providing unintended additional discounts. |
         | No | Allows multiple pricing rules to apply to the same product. This could result in stacking and providing multiple discounts applied to your listing price. |

         {:.bs-callout-warning}

>          To discard subsequent rules, a pricing rule must use your defined priorities which are set in the Priority field of each rule, and multiple rules should not have the same defined priority. See **Priority** in the **Add a New Rule** step.

Reviewer: ```suggestion
         To discard subsequent rules, a pricing rule must use the defined priorities that are set in the Priority field of each rule, and multiple rules should not have the same defined priority. See **Priority** in the **Add a New Rule** step.
```
